src/redux/actions/experiments/deleteMetadataTrack.js

```diff
@@ -35,14 +31,14 @@ const deleteMetadataTrack = (
     );
 
     dispatch({
-      type: PROJECTS_METADATA_DELETE,
+      type: EXPERIMENTS_METADATA_DELETE,
       payload: {
         key: metadataKey,
-        projectUuid,
+        projectUuid: experimentId,
```

Comment:
why don't you change this `projectUuid` to `experimentId`? Because the type is already changed

Reply:
I didn't want to change more things in the platform, so I consciously decided not to do that, but I could try to do it in this PR as well maybe, I'll try

Reply:
Ok, I removed proejctuuid from here and the other places in samples, a lot of added changes came from this